src/main/java/org/example/dao/PointDAOImpl.java

```diff
@@ -0,0 +1,80 @@
+package org.example.dao;
+
+import org.example.model.Point;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PointDAOImpl implements PointDAO {
+    private Connection connection;
+
+    public PointDAOImpl() {
+        try {
+            connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/DB", "username", "password");
```

Comment:
Why are we specifying all the config stuff here? Isn't that the purpose of the config?

Reply:
deleted